rollup/internal/controller/sender/sender.go

```diff
@@ -641,7 +641,8 @@ func (s *Sender) getBlockNumberAndBaseFeeAndBlobFee(ctx context.Context) (uint64
 		parentExcessBlobGas := eip4844.CalcExcessBlobGas(*header.ExcessBlobGas, *header.BlobGasUsed)
 		blobBaseFee = eip4844.CalcBlobFee(parentExcessBlobGas).Uint64()
```

Comment:
pls update to:
```
	if excess := block.ExcessBlobGas; excess != nil {
		blobBaseFee = eip4844.CalcBlobFee(*excess).Uint64()
	}
```
since the block fetched has been changed.

Reply:
Thanks, updated it.